### PR TITLE
Best chickoo_lassi for summer seassion. :)

### PR DESCRIPTION
**Pull request by Priyanka788**

### CODE REVIEW

File: src/recipes/chickoo_lassi.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            "ingredient": "<honey>"
+        },
+    ],
+    "directions": [
+        "<Step 1:-Place all the ingredients in a blender and blend well.>",
+        "<Step 2:-Sieve the lassi to avoid lumps and make it smooth.>",
+		"<Step 3:-Garnish with almond and more dry fruits.>",
+        "<Step 4:-Serve it cold.>"
+    ],
+    "image": "<Chickoo Lassi Recipe.jpg>",
```
Comment (alfg) [inline src/recipes/chickoo_lassi.json]:
```suggestion
    "image": "chickoo-lassi.jpg",
```

**Comment by Priyanka788** [comment]
If more changes required please tell me. :)

**Comment by alfg** [comment]
@Priyanka788 Ok, it's fine if you can't find one. More changes requested though.

File: src/recipes/chickoo_lassi.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "<Chickoo lassi>",
+    "description": "<A chilled glass of lassi is just the perfect way to refresh during the summer season.
+	                 Here's an easy and quick recipe to prepare delectable lassi at home with the flavours of chickoo fruit
+	                 blended with curd, milk,cardamom and honey>",
+    "github": "<Priyanka788>",
```
Comment (alfg) [inline src/recipes/chickoo_lassi.json]: Remove all braces from all fields.
```suggestion
    "github": "Priyanka788",
```

File: src/recipes/chickoo_lassi.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            "quantity": "<6>",
+            "measure": "<teaspon>",
+            "ingredient": "<honey>"
+        },
+    ],
+    "directions": [
+        Step 1:-Place all the ingredients in a blender and blend well.
+        Step 2:-Sieve the lassi to avoid lumps and make it smooth.
+        Step 3:-Garnish with almond and more dry fruits.
+        Step 4:-Serve it cold.
```
Comment (alfg) [inline src/recipes/chickoo_lassi.json]: These items still need to have double quotes to be valid JSON.

File: src/recipes/chickoo_lassi.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    "image": "<src/img/chickoo_lassi.jpg>",
+   
+    "keywords": [
+        "Chickoo",
+		"Dabur honey",
+		"Almond,Coconut,cherry,kismis",
```
Comment (alfg) [inline src/recipes/chickoo_lassi.json]: These can all be separate keywords.

**Comment by Priyanka788** [comment]
please check @alfg :)

File: src/recipes/chickoo_lassi.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            "ingredient": "honey"
+        },
+    ],
+    "directions": [
+       " Step 1:-Place all the ingredients in a blender and blend well.",
+       " Step 2:-Sieve the lassi to avoid lumps and make it smooth.",
+       "Step 3:-Garnish with almond and more dry fruits.",
+       "Step 4:-Serve it cold."
+    ],
+    "image": "src/img/chickoo_lassi.jpg",
```
Comment (alfg) [inline src/recipes/chickoo_lassi.json]:
```suggestion
    "image": "chickoo_lassi.jpg",
```

**Comment by alfg** [comment]
Thanks for the fixes. I added a few corrections on the JSON and have merged.

**Pull request merged**